### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by Manage0** [Uni-Sopron/termelesiranyitas-o22-valami#1]

**Comment by hegyhati** [comment]
No, meg tudtam nezni. Sajnalom, hogy ilyen keson kuldted be, mert lehetett volna ebbol jot csinalni, de jelen formajaban nem eros... 

Az oke, hogy kitalalsz egy heurisztikat arra, hogy pakolod le a dolgokat, de ebbol egy nagyon primitivet sikerult, a szabadsagot leszamitva mindent FCFS megprobal a deadline napjara tenni. A szabadsagos meg kicsit moho, mert elobb teszi be az elso napra, mint kesobbire. Szoval viszonylag keves munkabol ossze lehetett volna rakni egy merhetoen jobb heurisztikat: PTO-s fuggvenyedet hivni mindenre, es visszafele priorizalva a napokat. 

Tobbszor irod, hogy az algoritmusod helyes, de ez alatt nem tudom, mit ertesz. 

Ha van egy egyszeru heurisztika, akkor azt jo lett volna nagy adathalmazon letesztelni, illetve ket egyszeru heurisztikat osszehasonlitani ezen a nagy adathalmazon, melyik hogy teljesit.

Python latszik hogy nem a preferalt nyelved, sok helyen nem "pythonic" modon vannak megoldva dolgok. Ha erdekel, egyszer ugorj majd be, atbeszeljuk. De nyelvtol fuggetlenul is vannak "erdekes megoldasok".

A jegyen meg gondolkodom, de 3 fele hajlok.

**Comment by hegyhati** [comment]
Legyen 4, de kerlek a fenti heurisztikara alakitsd at, az az 1 kod legyen, a 4 eset meg igazabol csak 4 input json.
